CONTRIBUTING.md

```diff
@@ -53,12 +53,12 @@ freely where these tools come from on your local environment.
 
 ## Running tests
 
-Running `make` will perform all integration tests, check formatting and check for common shell problems in code.
+Running `make` will perform all integration tests, check formatting, and check for common shell problems in code.
```

Comment:
I don't know if we care about the Oxford comma or not, but I think most coders prefer it.